app/Nova/Actions/UdbOrganizer/ActivateUdbOrganizer.php

```diff
@@ -51,6 +52,8 @@ public function handle(ActionFields $fields, Collection $udbOrganizers): void
 
             if ($success) {
                 $this->udbOrganizerRepository->updateStatus($udbOrganizer->id, UdbOrganizerStatus::Approved);
+
+                UdbOrganizerApproved::dispatch($udbOrganizer->organizerId, $udbOrganizer->integrationId);
```

Comment:
This needs to be dispatched when the state of the `udbOrganizer` is set to `UdbOrganizerStatus::Approved`, not just inside the Nova action. So I would do the dispatch inside the repository or the model.

Reply:
You can also see that this is not the same as the already existing `RequestUdbOrganizer`